### PR TITLE
Camel case variables

### PR DESCRIPTION
**Pull request by kranjanford**

### CODE REVIEW

File: Example/openxcframework/Main.storyboard [hunk 1]
```diff
@@ -1754,15 +1755,15 @@
...
-                        <outlet property="tergetURLnamelabel" destination="lHN-0f-nsv" id="6hF-rF-HDm"/>
-                        <outlet property="traceURLname" destination="Hhi-gN-yQP" id="oSX-OD-Tn1"/>
-                        <outlet property="uploadtraceswitch" destination="VSn-sJ-ltf" id="Acz-vn-Xah"/>
+                        <outlet property="apiSourceNameLabel" destination="AHc-rV-ZGu" id="dv8-YK-ZdT"/>
+                        <outlet property="dweetNameField" destination="gc8-6y-klU" id="t5H-Wa-LCg"/>
+                        <outlet property="dweetNameLabel" destination="mcU-7m-8Hq" id="8CN-34-gxD"/>
+                        <outlet property="dweetSwitch" destination="K94-d9-L1s" id="3he-sk-beO"/>
+                        <outlet property="recordingNameField" destination="6tc-ZE-M8c" id="SWE-PI-bmF"/>
+                        <outlet property="recordingSwitch" destination="fJi-1d-iwc" id="eNF-9c-hbg"/>
+                        <outlet property="tergetUrlNameLabel" destination="lHN-0f-nsv" id="PdE-7O-Fy6"/>
```
Comment (mstocke) [inline Example/openxcframework/Main.storyboard:1764]: The variable name should start with "target", not "terget"

File: Example/openxcframework/RecordingSourceController.swift [hunk 2]
```diff
@@ -11,45 +11,46 @@ import openXCiOSFramework
...
-    @IBOutlet weak var tergetURLnamelabel: UILabel!
-    @IBOutlet weak var apiSourcenamelabel: UILabel!
+    @IBOutlet weak var recordingSwitch: UISwitch!
+    @IBOutlet weak var recordingNameField: UITextField!
+    @IBOutlet weak var dweetSwitch: UISwitch!
+    @IBOutlet weak var dweetNameField: UITextField!
+    @IBOutlet weak var dweetNameLabel: UILabel!
+    @IBOutlet weak var uploadTraceSwitch: UISwitch!
+    @IBOutlet weak var traceUrlNameField: UITextField!
+    @IBOutlet weak var tergetUrlNameLabel: UILabel!
```
Comment (mstocke) [inline Example/openxcframework/RecordingSourceController.swift:21]: As above, The variable name should start with "target", not "terget"

File: Example/openxcframework/RecordingSourceController.swift [hunk 3]
```diff
@@ -11,45 +11,46 @@ import openXCiOSFramework
...
-        traceURLname.addTarget(self, action: #selector(traceURLFieldDidChange), for: UIControl.Event.editingChanged)
-        traceURLname.addTarget(self, action: #selector(keyboardWillShow), for: UIControl.Event.editingDidBegin)
-        traceURLname.addTarget(self, action: #selector(keyboardWillHide), for: UIControl.Event.editingDidEnd)
-        traceURLname.isHidden = true
-        tergetURLnamelabel.isHidden = true
+        traceUrlNameField.addTarget(self, action: #selector(traceURLFieldDidChange), for: UIControl.Event.editingChanged)
+        traceUrlNameField.addTarget(self, action: #selector(keyboardWillShow), for: UIControl.Event.editingDidBegin)
+        traceUrlNameField.addTarget(self, action: #selector(keyboardWillHide), for: UIControl.Event.editingDidEnd)
+        traceUrlNameField.isHidden = true
+        tergetUrlNameLabel.isHidden = true
```
Comment (mstocke) [inline Example/openxcframework/RecordingSourceController.swift:53]: As above, The variable name should start with "target", not "terget".

File: Example/openxcframework/RecordingSourceController.swift [hunk 4]
```diff
@@ -59,22 +60,22 @@ class RecordingSourceController: UIViewController,UITextFieldDelegate {
         print(traceOutOn)
         // update UI if necessary
         if traceOutOn == true {
-            uploadtraceswitch.setOn(true, animated:false)
-            traceURLname.isHidden = false
-            tergetURLnamelabel.isHidden = false
+            uploadTraceSwitch.setOn(true, animated:false)
+            traceUrlNameField.isHidden = false
+            tergetUrlNameLabel.isHidden = false
```
Comment (mstocke) [inline Example/openxcframework/RecordingSourceController.swift:65]: The variable name should start with "target", not "terget".

File: Example/openxcframework/RecordingSourceController.swift [hunk 5]
```diff
@@ -168,21 +169,21 @@ class RecordingSourceController: UIViewController,UITextFieldDelegate {
...
+            dweetNameLabel.isHidden = true
         }
     }
     @IBAction func uploadTraceChange(_ sender: UISwitch) {
         UserDefaults.standard.set(sender.isOn, forKey:"uploadTaraceOn")
         if sender.isOn {
-            traceURLname.isHidden = false
-            tergetURLnamelabel.isHidden = false
+            traceUrlNameField.isHidden = false
+            tergetUrlNameLabel.isHidden = false
```
Comment (mstocke) [inline Example/openxcframework/RecordingSourceController.swift:183]: The variable name should start with "target", not "terget".

File: Example/openxcframework/RecordingSourceController.swift [hunk 6]
```diff
@@ -168,21 +169,21 @@ class RecordingSourceController: UIViewController,UITextFieldDelegate {
...
         } else {
-            traceURLname.isHidden = true
-            tergetURLnamelabel.isHidden = true
+            traceUrlNameField.isHidden = true
+            tergetUrlNameLabel.isHidden = true
```
Comment (mstocke) [inline Example/openxcframework/RecordingSourceController.swift:186]: The variable name should start with "target", not "terget".

File: Example/openxcframework/SendCanViewController.swift [hunk 7]
```diff
@@ -39,7 +38,7 @@ class SendCanViewController: UIViewController, UITextFieldDelegate {
         bm = BluetoothManager.sharedInstance
         
         // set default CAN target
-        vm.setCanDefaultTarget(self, action: SendCanViewController.default_sendcan_rsp)
+        vm.setCanDefaultTarget(self, action: SendCanViewController.defaultSendcanResponse)
```
Comment (mstocke) [inline Example/openxcframework/SendCanViewController.swift:41]: The variable name should be defaultSendCanResponse

File: Example/openxcframework/SendCanViewController.swift [hunk 8]
```diff
@@ -62,15 +61,15 @@ class SendCanViewController: UIViewController, UITextFieldDelegate {
         
     
     }
-    func default_sendcan_rsp(_ rsp:NSDictionary) {
+    func defaultSendcanResponse(_ rsp:NSDictionary) {
```
Comment (mstocke) [inline Example/openxcframework/SendCanViewController.swift:64]: As above, The variable name should be defaultSendCanResponse

File: Example/openxcframework/StatusViewController.swift [hunk 9]
```diff
@@ -74,35 +74,35 @@ class StatusViewController: UIViewController, UITableViewDelegate, UITableViewDa
...
              }
              else
              {
-                 self.disconnectBtn.isHidden = true
-                 self.searchBtn.isEnabled = true
-                 self.NetworkImg.isHidden = true
-                 self.actConLab.text = "---"
-                 self.msgRvcdLab.text = "---"
-                 self.searchBtn.setTitle("SEARCH FOR BLE VI",for:UIControl.State())
+                 self.disConnectButton.isHidden = true
```
Comment (mstocke) [inline Example/openxcframework/StatusViewController.swift:100]: As above, the variable name should be disconnectButton

File: Example/openxcframework/StatusViewController.swift [hunk 10]
```diff
@@ -410,39 +411,39 @@ class StatusViewController: UIViewController, UITableViewDelegate, UITableViewDa
...
+                    self.splitTraceButton.isHidden = false
+                    self.startStopButton.isHidden = false
+                    self.startStopButton.isSelected = true
                 }
-                self.disconnectBtn.isHidden = false
-                self.peripheralTable.isHidden = true
-                self.actConLab.text = "✅"
-                self.NetworkImg.isHidden = true
-                self.searchBtn.setTitle("BLE VI CONNECTED",for:UIControl.State())
+                self.disConnectButton.isHidden = false
```
Comment (mstocke) [inline Example/openxcframework/StatusViewController.swift:418]: As above, the variable name should be disconnectButton

File: Example/openxcframework/Main.storyboard [hunk 11]
```diff
@@ -576,20 +576,20 @@
...
-                        <outlet property="restartTraceBtn" destination="lZD-ZF-Ntt" id="T7G-uk-hDh"/>
-                        <outlet property="searchBtn" destination="pPj-fy-gM3" id="d2X-TH-VKG"/>
-                        <outlet property="splitTraceBtn" destination="k8a-rk-re3" id="XJN-kN-9zx"/>
-                        <outlet property="startStopBtn" destination="gYU-dO-wXU" id="IZH-Bl-BtX"/>
-                        <outlet property="throughputLab" destination="mEW-VZ-SbC" id="kUX-Ft-CAO"/>
-                        <outlet property="verLab" destination="ioC-fU-wIq" id="tW1-a0-pkJ"/>
+                        <outlet property="activeConnectionLabel" destination="twl-J0-AhF" id="eyc-LF-AWS"/>
+                        <outlet property="averageMessageLabel" destination="8a9-sK-hBj" id="4kL-AF-pUa"/>
+                        <outlet property="deviceIdLabel" destination="XOY-Ud-hZJ" id="Mjy-m2-rwq"/>
+                        <outlet property="disConnectButton" destination="ceN-pV-EP6" id="muq-GJ-9kr"/>
```
Comment (mstocke) [inline Example/openxcframework/Main.storyboard:582]: As above, the variable name should be disconnectButton

File: Example/openxcframework/StatusViewController.swift [hunk 12]
```diff
@@ -10,28 +10,28 @@ import openXCiOSFramework
...
+    @IBOutlet weak var networkImageView: UIImageView!
     
-    fileprivate var troughputLoop: Timer!
+    fileprivate var throughPutLoop: Timer!
     // scan/connect button
-    @IBOutlet weak var searchBtn: UIButton!
+    @IBOutlet weak var searchButton: UIButton!
     // disconnect button
-    @IBOutlet weak var disconnectBtn: UIButton!
+    @IBOutlet weak var disConnectButton: UIButton!
```
Comment (mstocke) [inline Example/openxcframework/StatusViewController.swift:26]: As above, the variable name should be disconnectButton

File: Example/openxcframework/StatusViewController.swift [hunk 13]
```diff
@@ -45,9 +45,9 @@ class StatusViewController: UIViewController, UITableViewDelegate, UITableViewDa
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.disconnectBtn.isHidden = true
-        self.splitTraceBtn.isHidden = true
-        self.startStopBtn.isHidden = true
+        self.disConnectButton.isHidden = true
```
Comment (mstocke) [inline Example/openxcframework/StatusViewController.swift:48]: As above, the variable name should be disconnectButton

File: Example/openxcframework/StatusViewController.swift [hunk 14]
```diff
@@ -74,35 +74,35 @@ class StatusViewController: UIViewController, UITableViewDelegate, UITableViewDa
...
-                  self.restartTraceBtn.isHidden = true
+                  self.restartTraceButton.isHidden = true
              }
     }
     func bluetoothCheck(){
         if (bm.isBleConnected) {
                  DispatchQueue.main.async {
-                     self.disconnectBtn.isHidden = false
-                     self.NetworkImg.isHidden = true
+                     self.disConnectButton.isHidden = false
```
Comment (mstocke) [inline Example/openxcframework/StatusViewController.swift:93]: As above, the variable name should be disconnectButton

File: Example/openxcframework/StatusViewController.swift [hunk 15]
```diff
@@ -562,17 +564,17 @@ class StatusViewController: UIViewController, UITableViewDelegate, UITableViewDa
...
-                       self.startStopBtn.isHidden = true
-                       self.throughputLab.text = "---"
-                       self.averageMessageLab.text = "---"
+                       self.activeConnectionLabel.text = "---"
+                       self.messagesReceivedLabel.text = "---"
+                       self.versionLabel.text = "---"
+                       self.deviceIdLabel.text = "---"
+                       self.platformLabel.text = "---"
+                       self.searchButton.setTitle("SEARCH FOR BLE VI",for:UIControl.State())
+                       self.disConnectButton.isHidden = true
```
Comment (mstocke) [inline Example/openxcframework/StatusViewController.swift:573]: As above, the variable name should be disconnectButton

File: openxcframework/BluetoothManager.swift [hunk 16]
```diff
@@ -39,27 +39,27 @@ open class BluetoothManager: NSObject,CBCentralManagerDelegate,CBPeripheralDeleg
   // config for outputting debug messages to console
   fileprivate var managerDebug : Bool = false
   var result: String!
-  var troughputtimer: Timer!
+  var troughputTimer: Timer!
```
Comment (mstocke) [inline openxcframework/BluetoothManager.swift:42]: The variable name should be throughputTimer

File: Example/openxcframework/DataSourceController.swift [hunk 17]
```diff
@@ -118,20 +118,20 @@ class DataSourceController: UIViewController,UITextFieldDelegate,CLLocationManag
...
             interfaceValue = vehicleInterface
             
             
         }
             
-        else if vehicleInterface == prerecordTrace {
+        else if vehicleInterface == preRecordTrace {
             if let tracefile = (UserDefaults.standard.value(forKey: "traceInputFilename")  as? String){
-                playname.text = tracefile
+                TraceFilePlayNameField.text = tracefile
```
[the remark this answers is not in the document]
Comment (kranjanford) [reply Example/openxcframework/DataSourceController.swift:132]: done

File: Example/openxcframework/RecordingSourceController.swift (hunk 2 above)
Comment (kranjanford) [reply Example/openxcframework/RecordingSourceController.swift:21]: done

File: Example/openxcframework/SendCanViewController.swift (hunk 7 above)
Comment (kranjanford) [reply Example/openxcframework/SendCanViewController.swift:41]: done but this is function name .

File: Example/openxcframework/SendCanViewController.swift (hunk 8 above)
Comment (kranjanford) [reply Example/openxcframework/SendCanViewController.swift:64]: Function Name done

File: Example/openxcframework/StatusViewController.swift (hunk 12 above)
Comment (kranjanford) [reply Example/openxcframework/StatusViewController.swift:26]: Done

File: openxcframework/BluetoothManager.swift (hunk 16 above)
Comment (kranjanford) [reply openxcframework/BluetoothManager.swift:42]: done

**Pull request merged**